src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -40,10 +45,12 @@
             </div>
           </div>
         </ng-container>
-        &nbsp;
         <div class="field">
           <div *ngIf="!f.valid" class="error">
-            <sup>*</sup> Field can't left blank.
+            <sup>*</sup> Field can't be left blank.
+          </div>
+          <div *ngIf="!validRange" class="error">
+            <sup>*</sup> Field size out of range.
```

Comment:
-  Value should be in range <min>-<max>
-  Value should not be less than <min>
-  Value should not be greater than <max>